frontend/src/Docs/DocsSearch.css

```diff
@@ -0,0 +1,53 @@
+.search-box {
+    position: relative;
+  }
+
+  .search-box input {
+    width: 95%;
+    height: 1.5em;
+    font-size: 20px;
+    border-radius: 3px;
+  }
+  
+  
+  .suggestions {
+    position: absolute;
+    top: 100%;
+    left: 0;
+    right: 0;
+    background-color: var(--impressed);
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    border: 1px solid gray;
+    z-index: 1;
+    display: block;
+  }
+  
+  .suggestions li {
+    cursor: pointer;
+    color: var(text);
+  }
+  
+  .suggestions li:hover {
+    background-color: var(--bg);
+  }
+
+  .link {
+    display: block;
+    width: 100%;
+    height: 100%;
+    padding: 10px;
+  }
+
+  #input {
```

Comment:
worried that, due to by own hubris and terrible mistakes w.r.t. CSS / React, this might conflict somewhere else. Might change the id to something like "docs-search-input" instead but fine for now